dev.Dockerfile

```diff
@@ -37,7 +37,15 @@ COPY docker/dev/Settings-dev.toml /app/Settings.toml
 COPY docker/dev/service.sh /etc/service/geoengine/run
 RUN chmod +x /etc/service/geoengine/run \
     && \
-    adduser --disabled-password --gecos "" geoengine
+    adduser --disabled-password --gecos "" geoengine 
+
+# Create upload folder and chown it to the geoengine user
+RUN mkdir -p /app/upload \
+    && \
+    chown -R geoengine:geoengine /app/upload \
```

Comment:
what happens if you mount a host folder/volume under this path (like we did it in the terranova demo for the data definitions)? Will it try to chown the host folder? Please test it.

Reply:
this is run at image creation. it will not change the folder you mount inside. however, it will create new folder inside the folder with the uid of the user in the container. so uid mapping would be a good idea.